contents/docs/deployment/snippet-installation.md

```diff
@@ -21,11 +21,9 @@ Having copied the snippet from the Settings page, you should now paste this with
 
 We recommend adding the PostHog snippet both on your homepage and your application (if you have one). That means you'll be able to follow a user from the moment they come onto your website, all the way through Sign Up and their actual usage of your product.
 
-## Adding Multiple URLs 
+## Adding Multiple Domains 
 
-PostHog supports tracking across multiple URLs and domains. Also on your Project Settings page, you are able to add more URLs that you wish to track. 
-
-![Snippet Settings Screenshot](../../images/features/snippet/add-url.png)
+PostHog supports tracking across multiple domains - just include the same snippet on all the websites you wish to track under the same project.
```

Comment:
Q: Should we also mention permitted domains here?

Reply:
Yeah that's a good point.